### PR TITLE
cas datadir

### PR DESCRIPTION
**Pull request by pmauduit**
This commit is the datadirization diff between my branch (georchestra-datadir) and master, specifically for the cas module.

Warning: this commit won't be complete unless an underlying PR on the template configuration is merged. See https://github.com/georchestra/template/pull/20. It also requires the one addressing the georchestra-commons module insertion (https://github.com/georchestra/georchestra/pull/1035).

TODO:
* [x] compilation
* [x] testsuite
* [x] runtime tests (with `georchestra.datadir`)
* [x] runtime tests (without `georchestra.datadir`)

### CODE REVIEW

File: cas-server-webapp/pom.xml [hunk 1]
```diff
@@ -134,12 +148,145 @@
...
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>net.sf.debian-maven</groupId>
+            <artifactId>debian-maven-plugin</artifactId>
+            <version>1.0.6</version>
+            <configuration>
+              <packageName>georchestra-cas</packageName>
+              <packageDescription>Debian package for the geOrchestra's CAS server</packageDescription>
```
Comment (fvanderbiest) [inline cas-server-webapp/pom.xml]: Debian package for geOrchestra CAS server

File: cas-server-webapp/pom.xml [hunk 2]
```diff
@@ -134,12 +148,145 @@
...
+              <projectOrganization>geOrchestra</projectOrganization>
+              <maintainerName>Pierre Mauduit</maintainerName>
+              <maintainerEmail>pierre.mauduit@gmail.com</maintainerEmail>
```
Comment (fvanderbiest) [inline cas-server-webapp/pom.xml]: PSC / psc@georchestra.org

File: cas-server-webapp/pom.xml [hunk 3]
```diff
@@ -134,12 +148,145 @@
...
+      <id>rpmPackage</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-war-plugin</artifactId>
+            <version>2.1.1</version>
+            <configuration>
+              <classifier>generic</classifier>
+              <packagingExcludes>**/docs/**,**/openlayers/examples/**,**/openlayers/tests/**</packagingExcludes>
```
Comment (fvanderbiest) [inline cas-server-webapp/pom.xml]: do we really need these excludes in CAS ?

**Comment by fvanderbiest** [comment]
LGTM otherwise

File: cas-server-webapp/pom.xml (hunk 1 above)
Comment (pmauduit) [inline cas-server-webapp/pom.xml]: done

File: cas-server-webapp/pom.xml (hunk 2 above)
Comment (pmauduit) [inline cas-server-webapp/pom.xml]: done

File: cas-server-webapp/pom.xml (hunk 3 above)
Comment (pmauduit) [inline cas-server-webapp/pom.xml]: removed

**Pull request merged**